### PR TITLE
Determine glyph type

### PR DESCRIPTION
**Pull request by kalapi**
This commit adds sub-directory with subsets of the '.nam' files. The subsets are divided by glyph types with regards to the whether a glyph is 'outline/drawn', 'composite', or 'either'.

### CODE REVIEW

File: tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Glyph Type Sorting
```
Comment (davelab6) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/README.md]: I suggest just appending this content to the parent directory README

**Comment by kalapi** [comment]
I can confirm that the sender (of this pull request) and the commit author are the same entity. I have edited my `git config user.email` to reflect the details in the CLA.

File: tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/EXPERT_composites_207.txt [hunk 2]
```diff
@@ -0,0 +1,207 @@
+0x2153  ⅓ onethird
+0x2154  ⅔ twothirds
+0x215B  ⅛ oneeighth
+0x215C  ⅜ threeeighths
+0x215D  ⅝ fiveeighths
+0x215E  ⅞ seveneighths
+0x2191  ↑ upArrow
+0x2192  → rightArrow
+0x2193  ↓ downArrow
+          aacute.sc
```
Comment (davelab6) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/EXPERT_composites_207.txt]: I don't think the smallcaps glyphs are composites; they need to be drawn (using 'powertools' like RMX, yes, but still, not composites... :)

Comment (davelab6) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/EXPERT_composites_207.txt]: Ah, this is all accents .sc glyphs. oops =)

File: tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/README.md (hunk 1 above)
Comment (kalapi) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/README.md]: Ok, I will remove it from the sub-directory for the time being and find a way to incorporate it into the parent directory's readme. Thanks!

**Comment by davelab6** [comment]
@kalapi thanks :) I updated the contributing docs with a tip for this, https://github.com/googlefonts/gf-docs/commit/7a520b44cd7635dfd6f7ad6041afe72f92dc23e7

**Comment by kalapi** [comment]
Haha! Sorry about that! :)

**Comment by alexeiva** [comment]
The new list doesn't include caronalt > caroncomb.alt fix, possibly something other.

**Comment by davelab6** [comment]
@alexeiva please fix :)

**Comment by alexeiva** [comment]
I fixed it in #302. It was the only problem.

**Pull request merged**